### ISSUE TITLE
Problem: multi-lines separated with string

### ISSUE THREAD

**Issue by emisjerry** [trey-wallis/obsidian-note-splitter#10]
Here's my origianl note, the delimiter is `###`:

```
Question 1.

Question 1 description.

---
Answer of question 1.

###
Question 2.

Question 2 description.

---
Answer of question 2.
```

When i checked on "Using content as title", the plugin responsed error of creating file. It seems like that the whole content become the title, not the first sentence.

It should creates "Question 1.md" and "Question 2.md".

Thanks.

**Comment by decaf-dev** [comment]
Thanks for finding this. The markdown you posted was helpful in reproducing the issue.

**Comment by decaf-dev** [comment]
Fixed in 1.1.1
